.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,8 +7,9 @@
 
 Please ensure you've completed all of the following.
 
-- [ ] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
-- [ ] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
+- [ ] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
```

Comment:
Please undo the 1st two lines of changes. Though the issues get reported in the main (ferdium-app) repo, the contriuting guidelines and scripts to run are different between the 2 repos.

Reply:
I understand what you mean but the url on the 1st line does not work. I used the CONTRIBUTING.md doc to install repo recipes on my local. 
Should we create a new CONTRIBUTING.md ?

Reply:
@Mralexandresys is right, the link to https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md is broken.

Maybe I fired too quicky in merging https://github.com/ferdium/ferdium-recipes/pull/471, @vraravam let us know if something has to be updated.